### PR TITLE
feat: add feature toggle for new information architecture

### PR DESCRIPTION
**Pull request by duidae**

### CODE REVIEW

File: packages/core/src/constants/feature-flag.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const enableNewInfoArch = false
```
Comment (Aylie-Chou) [inline packages/core/src/constants/feature-flag.js:1]: Could we also export `enableNewInfoArch ` in order to use `import { enableNewInfoArch } from xxx` syntax ?

Comment (duidae) [reply packages/core/src/constants/feature-flag.js:1]: okok

Comment (duidae) [reply packages/core/src/constants/feature-flag.js]: fixed in https://github.com/twreporter/twreporter-npm-packages/pull/302/commits/32d1633844abd9e57217e7b959c74621a24f11fc

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**